### DISCUSSION TITLE
视频播放页拉到最下面显示几个空白框。。。

### DISCUSSION THREAD

**Discussion by drw2005** [the1812/Bilibili-Evolved#3667] [提问 / Q&A]
视频播放页拉到最下面显示如下图，是哪里设置的不对吗？
![无标题](https://user-images.githubusercontent.com/33473981/189030091-ba7f9a64-8ffd-43f6-9a40-8d0a1e3f1508.png)

**Comment by the1812** [comment]
没见过, 应该跟脚本没关系吧

**Reply by drw2005** [reply]
我关闭脚本就没这个问题了。
 Chromium 内核版本号：103

**Reply by the1812** [reply]
只开关脚本看不出来是哪个功能的, 开关脚本里面的功能找找看是哪个导致的
